code/__DEFINES/dcs/flags.dm

```diff
@@ -22,6 +22,8 @@
 /// Causes all detach arguments to be passed to detach instead of only being used to identify the element
 /// When this is used your Detach proc should have the same signature as your Attach proc
 #define ELEMENT_COMPLEX_DETACH (1 << 2)
+/// For machines and structures that should not break into parts, eg, holodeck stuff
+#define NODECONSTRUCT_1	(1<<7)
```

Comment:
This seems to be lacking any supporting code to make it work, and also seems to conflict with `PREVENT_CONTENTS_EXPLOSION_1`
https://github.com/tgstation/tgstation/pull/80104 is probably relevant here